server/chat-commands/info.ts

```diff
@@ -2644,6 +2645,27 @@ export const commands: ChatCommands = {
 	},
 	showhelp: [`/show [url] - shows an image or video url in chat. Requires: whitelist % @ # &`],
 
+	regtime: 'registertime',
+	async registertime(target, room, user) {
+		this.checkChat();
+		this.runBroadcast();
+		if (!user.autoconfirmed) return this.errorReply(`Only autoconfirmed users can use this command.`);
+		target = toID(target);
+		if (!target) target = user.id;
+		const raw = await Net(`https://${Config.routes.root}/users/${target}.json`).get().catch(() => {});
+		if (!raw) return this.errorReply(`An error occurred when retrieving user data.`);
```

Comment:
I think it's pretty bad form to refuse to say what the error was. This is one of the design principles that's in CONTRIBUTING now.